opentracing-api/pom.xml

```diff
@@ -29,8 +29,8 @@
 
     <properties>
         <main.basedir>${project.basedir}/..</main.basedir>
-        <main.java.version>1.6</main.java.version>
-        <main.signature.artifact>java16</main.signature.artifact>
+        <main.java.version>1.7</main.java.version>
```

Comment:
People come to zipkins gitter all the time asking to make sure `brave` is 1.6 compatible. I'd say its still unfortunately a hard requirement for promoting adoption